include/shellstrings.h

```diff
@@ -52,33 +75,43 @@ void shell_save();
  * Returns:
  * 	- None.
  */
- void shell_print(char** lines);
+ void shell_print(char **lines);
 
+/*
+ * shell_input - Input file confirmation prompt for the shell.
+ *
+ * Parameters:
+ *  - input_file: String of the file input by the user.
+ *  - status: String for the file type - dictionary or 
+ *              this is a string because shell_input is used
+ *              to print a confirmation message on the screen
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_input(char *filename, char *status, bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!

Reply:
changed!